lib/Dialects/LinalgTransform/IR/TransformOpInterface.cpp

```diff
@@ -104,9 +104,12 @@ void transform::TransformState::updatePayloadOps(
 
 LogicalResult
 transform::TransformState::applyTransform(TransformOpInterface transform) {
+  TimingScope timing = tm.getRootScope();
+  TimingScope transformOp = timing.nest(transform->getName().getStringRef());
```

Comment:
What happens when there's more than one op with the same name (double tiling is a thing, so is having multiple sequences based on different matches)?

Reply:
If they have the same name they end up in the same bucket. For example, the report for a double tiling will report only one 'tiling' entry where the value is the sum of the execution of the two tiling op. We can fix this by appending location information.